.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: go
-go:       1.11
+go:       1.13
```

Comment:
Note to self:
"This PR require at least go1.13"

> Per the number literal proposal, Go 1.13 supports a more uniform and modernized set of number literal prefixes.

https://golang.org/doc/go1.13#language